### PR TITLE
Updates to polish gliders for consistency

### PR DESCRIPTION
**Pull request by Scumi**
I've recognized a bunch of inconsistencies with most of the polish gliders, which I tried to correct. The original list looks like its based on the DMSt index list. Some of the changes should also be corrected there, but that's a different story.

Happy to discuss every changed line and ready to split-up the changes if necessary.

### CODE REVIEW

**Comment by moldhouse** [comment]
Thank you for fixing these inconsistencies. Indeed I would need some more explanation as I am not too familiar with polish gliders. I think we should split the updates on the aircraft naming and the updates on the manufacturer naming into two separate commits. Also I have mixed feelings about updating names for 'Jantar' and 'Diana' as these are the common names they are known by in most countries.

File: gliderlist.csv [hunk 1]
```diff
@@ -130,7 +130,7 @@ ID,Glider,Model,Manufacturer,Competition Class,Winglets,Double Seater,FES,Engine
 129,D 36,D 36,Akaflieg Darmstadt,18,,,,,108,108,108,108,108,108
 130,Diamant 18,Diamant 18,FFA,18,,,,,108,108,108,108,108,108
 131,Mü 26,Mü 26,Akaflieg München,18,,,,,102,102,102,102,102,102
-132,SZD-39 Cobra 17m,Cobra 17m,SZD,18,,,,,100,100,100,100,100,100
+132,SZD-39 Cobra 17,SZD-39 Cobra 17,PZL Bielsko,18,,,,,100,100,100,100,100,100
```
Comment (moldhouse) [inline gliderlist.csv:133]: Changes to Cobra look good

File: gliderlist.csv [hunk 2]
```diff
@@ -473,7 +473,7 @@ ID,Glider,Model,Manufacturer,Competition Class,Winglets,Double Seater,FES,Engine
 472,LS 7,LS 7,Rolladen-Schneider,Standard,,,,,107,107,107,107,107,107
 473,LS 7,LS 7 WL,Rolladen-Schneider,Standard,x,,,,107,107,107,107,107,107
 474,LS 7,LS 7 neo,Rolladen-Schneider,Standard,,,,,107,107,107,107,107,107
-475,SZD-55,SZD-55,PZL Bielsko,Standard,,,,,106,106,106,106,106,106
+475,SZD-55,SZD-55,Allstar PZL Glider,Standard,,,,,106,106,106,106,106,106
```
Comment (moldhouse) [inline gliderlist.csv:476]: So there exist PZL Bielsko, PZL Glider, Avionic and Allstar?

Comment (Scumi) [reply gliderlist.csv:476]: Yes, its unfortunately not that straightforward. There is a whole [Wiki](https://de.wikipedia.org/wiki/Szybowcowy_Zak%C5%82ad_Do%C5%9Bwiadczalny) article about this. It kind of depends on what you want to show: Type Certificate (TC) holder or manufacturer.

- [Allstar PZL Glider](https://szdallstar.com/de/kontakt/) is the current manufacturer and TC holder for some of those aircraft
- [Avionic](http://www.dianasailplanes.com/) has taken over the Diana 2 TC and manufacturing
- Everything else is not manufactured anymore (the "old" stuff) and the TC holder changes from time to time. I suggest using PZL Bielsko for all these.

**Comment by Scumi** [comment]
> Also I have mixed feelings about updating names for 'Jantar' and 'Diana' as these are the common names they are known by in most countries.

Yes, I understand that. Unfortunately there are 3 Jantar Standard versions and one of them has a different fuselage, which according to your reasoning needs a separate entry. Their "real" designation (SZD-41 / SZD-48) reflect that. The club class IGC index list also has a different index for those two.
Question: Does it really matter when your search algorithm can pull up both (and the open class ones) entries when typing in "Jantar"?

The SZD-59 Acro is even weirder in the current index list. It does only exist without winglets in its Acro (13.2m) configuration, but that should for sure not have an index of 100.

**Comment by moldhouse** [comment]
Yes I think you are right and we should go with the full name. I had a look at the type certificate and I think we should go with

SZD-41 Jantar Std.
SZD-48 Jantar Std. 2
SZD-48-3 Jantar Std. 3

and also add WL variants for all of them.

**Pull request merged**